routers/web/shared/actions/runners.go

```diff
@@ -313,6 +317,22 @@ func RunnerDeletePost(ctx *context.Context) {
 	successRedirectTo := rCtx.RedirectLink
 	failedRedirectTo := rCtx.RedirectLink + url.PathEscape(ctx.PathParam("runnerid"))
 
+	exist, err := db.Exist[actions_model.ActionTask](ctx, builder.Eq{"`runner_id`": runner.ID}.And(builder.In("`status`", actions_model.StatusWaiting, actions_model.StatusRunning, actions_model.StatusBlocked)))
```

Comment:
Since there is no pause feature for the runner, it's difficult to know when the runner is idle for a busy site.